src/unistd.rs

```diff
@@ -2031,6 +2031,18 @@ pub enum PathconfVar {
     /// queue; therefore, the maximum number of bytes a conforming application
     /// may require to be typed as input before reading them.
     MAX_INPUT = libc::_PC_MAX_INPUT,
+    #[cfg(any(
+        solarish,
+        freebsdlike,
+        target_os = "netbsd",
+    ))]
+    /// If a file system supports the reporting of holes (see lseek(2)),
+    /// pathconf() and fpathconf() return a positive number that represents the
+    /// minimum hole size returned in bytes.  The offsets of holes returned will
+    /// be aligned to this same value.  A special value of 1 is returned if the
+    /// file system does not specify the minimum hole size but still reports
+    /// holes.
+    MIN_HOLE_SIZE = libc::_PC_MIN_HOLE_SIZE,
```

Comment:
Seems that this is also supported by macOS, though this constant hasn't been added to libc. Anyway, let's merge this PR first:)

https://keith.github.io/xcode-man-pages/pathconf.2.html#_PC_MIN_HOLE_SIZE